docs/tables/mongodbatlas_advanced_cluster.md

```diff
@@ -0,0 +1,16 @@
+# Table: mongodbatlas_advanced_cluster
+
+MongoDB Atlas Cluster is a NoSQL Database-as-a-Service offering in the public cloud (available in Microsoft Azure, Google Cloud Platform, Amazon Web Services).
+
+## Examples:
+
+### List all MongoDB Advanced Clusters in the Project
```

Comment:
```suggestion
### List all MongoDB advanced clusters
```

We should not capitalize resource names/types, unless the provider capitalizes them in non-titles their documentation as per https://steampipe.io/docs/develop/table-docs-standards#style-conventions. For instance, in https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/#select-a-database-deployment-type:
> Select a database deployment type.
Begin the cluster configuration process by choosing your database deployment type. You can deploy a shared cluster or a dedicated cluster from this page. You can also deploy a serverless instance. Each panel leads to a configuration page where you can customize your database deployment to your specifications.

So it seems like MongoDB Atlas docs do not capitalize resource names, except in titles

Also, we don't need to say "in the project" since that's always the case